packages/gatsby-remark-images/src/index.js

```diff
@@ -171,17 +172,17 @@ module.exports = (
     const showCaptions = options.showCaptions && node.title
     let rawHTML = `
   <span
-    class="gatsby-resp-image-wrapper"
+    class="${imageWrapperClass}"
     style="position: relative; display: block; ${
       showCaptions ? `` : options.wrapperStyle
     } max-width: ${presentationWidth}px; margin-left: auto; margin-right: auto;"
   >
-    <span
-      class="gatsby-resp-image-background-image"
-      style="padding-bottom: ${ratio}; position: relative; bottom: 0; left: 0; background-image: url('${
-      fluidResult.base64
-    }'); background-size: cover; display: block;"
-    >${imageTag}</span>
+    <img
+      class="${imageBackgroundClass}"
+      style="width: 100%; height: 100%; position: relative; bottom: 0; left: 0; transition-delay: 0.5s; tranistion: opacity 0.5s; opacity: 1; background-size: cover; display: block;"
```

Comment:
typo in `transition` property

Reply:
I'm also not sure if it's good idea changing span to img here - there might be a11y implications if screenreader see 2 image tags instead of one for essentially same image?

Reply:
Hmmm... not sure to be honest I'll take your guidance on this one. Looking at `gatsby-image` it seems that the actual image is wrapped in a Tag that's passed in as a prop. So would we put this back into the span and keep the background as is? As this would be similar to what's going on there.

Reply:
From memory it was while I was working through a good way to implement it so went through several variations so I think this is probably just a side effect. I'll add it back in so essentially the below?

```
<img class="${imageBackgroundClass}" ... />
<span>${imageTag}</span>
```

Reply:
so previously it was
```html
<span class="gatsby-resp-image-wrapper">
  <span class="gatsby-resp-image-background-image">
    <img>
  </span>
</span>
```
which won't work if we won't to hide placeholder that previously was in `.gatsby-resp-image-background-image` (as it would hide img too) maybe this:
```html
<span class="gatsby-resp-image-wrapper">
  <span class="gatsby-resp-image-background-image"></span>
  <img>
</span>
```
to change as little as possible (limit potential regressions) in this PR?

Reply:
I'll have a play around to see what works 👍 thanks for comments, appreciate it :)